src/room_member.cpp

```diff
@@ -256,12 +256,24 @@ bool RoomMember::IsConnected() const {
     return state == State::Joining || state == State::Joined;
 }
 
+void RoomMember::CloseConnection() {
+    peer->Shutdown(300);
+    member_information.clear();
+    nickname.clear();
+    mac_address = {};
+    room_information = {};
+
+    chat_queue.clear();
```

Comment:
Now that I think about it, it might not be too bad to clear the chat instantly..
If we ever send the playerId instead of the nickname for chat messages we couldn't display them anymore properly because the MemberList will be empty already.

Let's keep this code for now and create an issue so we can consider solutions for the future?